### DISCUSSION TITLE
Livewire 3.4.11 bug: can access private/protected property in blade

### DISCUSSION THREAD

**Discussion by firer777** [livewire/livewire#8390] [7. Bugs (archived)]
Example I defined private property 
class Demo extends Component 
{
     private $post;
    public function mount(Post $post) 
   {
         $this->post = $post;
   }
}
In blade view: i can access and display value of private property $post by  {{ $this->post->name }}

**Comment by danie-ramdhani** [comment]
In V3, you need to use locked attribute https://livewire.laravel.com/docs/properties#locking-the-property

**Reply by firer777** [reply]
No lock use when avoid change property. Issue in private/protected property can display value outside component on blade by using $this-> post->name
